Not so fast... (More stability for the commitment tracker, gas price configuration improvements) (#2445)
nucypher/nucypher: docs/source/support/faq.rst

There is one setting only – 6 month duration, restake on, winddown on – for all CoinList participants.

See NuCypher coinlist for more details: `CoinList <https://coinlist.co/asset/nucypher>`_

> Q: How does my worker node chooses what price to use? Can I control this?

Reviewer: ```suggestion
Q: How does my worker node choose what price to use for transactions? Can I control this?
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

When sending a transaction, your worker node automatically chooses the gas price
using a `gas strategy <https://web3py.readthedocs.io/en/stable/gas_price.html>`_.

We currently offer three types of gas strategies,
based on the approximate confirmation time:

- ``slow``: Confirmation expected within **one hour**.
- ``medium``: Confirmation expected within **five minutes**.
- ``fast``: Confirmation expected within **one minute**.

Note that the times are an approximation, and the confirmation time may vary
when gas prices experience more volatility.

> In such situations, transactions may not be mined on the expected time;
> your node keeps track of the transactions and is capable of automatically

Reviewer: ```suggestion
In such situations, transactions may not be mined within the expected time.
However, your node keeps track of the transactions and is capable of automatically
```

sending replacement transactions to adjust to a scenario with new prices.
In any case, we recommend that you monitor your node to be sure that the
transactions are being sent and confirmed correctly.

You can set a gas strategy using the Ursula configuration command.
For example, the following command sets the ``medium`` gas strategy:

.. code:: bash

    (nucypher)$ nucypher ursula config --gas-strategy medium

There's an additional, advanced control mechanism for limiting the maximum
gas price that your node can spend.

> The ``NUCYPHER_MAX_GAS_PRICE_GWEI`` environment variable can be used to express,
> in `gwei`, such limit.

Reviewer: ```suggestion
The ``NUCYPHER_MAX_GAS_PRICE_GWEI`` environment variable, expressed
in `gwei`, can be used to specify this limit.
```

Reviewer 2: :heavy_check_mark:

This is complementary to the gas strategy that you may have configured.

.. warning::

>     If you set a maximum limit and the gas prices are high or very close
>     to that limit, it's possible that your node doesn't get
>     the commitment transaction included in the blockchain.

Reviewer: Unsure what you mean by "very close"...

```suggestion
    If you set a maximum limit and gas prices remain higher
    than that limit, it's possible that your node will not get
    the commitment transaction included in the blockchain.
```

    This implies that you can miss some rewards.

.. warning::

>     The maximum gas price limit is an experimental feature and may change or
>     be removed in the future.

Reviewer: ```suggestion
    The maximum gas price limit is an experimental feature and may be changed
    removed in the future.
```
